email.go

```diff
@@ -100,3 +101,27 @@ func (s *EmailService) SendBatch(emailRequests []*Email) ([]*EmailResponse, *Res
 
 	return email, response, nil
 }
+
+// SendBatchTemplated will build and execute request to send batch templated emails via the API.
+func (s *EmailService) SendBatchTemplated(emailRequests []*Email) ([]*EmailResponse, *Response, error) {
+	if len(emailRequests) < 1 {
+		return nil, nil, errors.New("You must pass a minimum of one email to SendBatch()")
+	}
+
+	var formatEmails map[string]interface{} = map[string]interface{}{
+		"Messages": emailRequests,
+	}
+
+	request, err := s.client.NewRequest("POST", emailWithBatchTemplateAPIPath, formatEmails)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	email := []*EmailResponse{}
+	response, err := s.client.Do(request, email)
```

Comment:
See comment left on PR. But the `tldr;` is we will never get an error back here due to this endpoint always returning `200`.